### PR TITLE
Trenches

### PR DESCRIPTION
**Pull request by Grey-Soldierman**
Ports 2 A2 trenches; Envelope Small & Envelope Big. 
Adds an entrenching tool based off an A2 shovel from the Land Rover. 
Someone smarter than me could combine the two to be able to deploy the trenches using the entrenching tool.

### CODE REVIEW

**Comment by Glowbal** [comment]
I will handle the scripting for this. Thanks @Grey-Soldierman

**Comment by Glowbal** [comment]
Also - we need to either move the models into `APL` or license the `data` folder in here APL.

File: AUTHORS.txt [hunk 1]
```diff
@@ -101,3 +101,4 @@ Valentin Torikian <valentin.torikian@gmail.com>
 VyMajoris(W-Cephei)<vycanismajoriscsa@gmail.com>
 Winter <simon@agius-muscat.net>
 zGuba
+Grey-Soldierman <gungamer101@hotmail.com>
```
Comment (esteldunedain) [inline AUTHORS.txt]: Add your name in alphabetical order please

**Comment by Glowbal** [comment]
Continue on #2104

**Pull request closed without merging**